gdal/ogr/ogrsf_frmts/sqlite/ogrsqliteutility.h

```diff
@@ -36,26 +36,32 @@
 
 #include <set>
 #include <string>
+#include <memory>
 
-typedef struct
+class SQLResult
 {
-    char** papszResult;
-    int nRowCount;
-    int nColCount;
-    char *pszErrMsg;
-    int rc;
-} SQLResult;
+    public:
+        SQLResult(char** papszResult, int nRow, int nCol);
+        ~SQLResult ();
+
+        int         RowCount() const;
```

Comment:
a inline definition for RowCount() and ColCount() would be more appropriate

Reply:
True, fixed